FoodOrdering.html

```diff
@@ -0,0 +1,172 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Food Ordering System</title>
+  <link href="./resources/css/FoodOrdering.css" type="text/css" rel="stylesheet">
+</head>
+<body>
+    <div class="top-section">
+        <header>
+            <div class="logo">
+                <img src="./resources/images/logo_footer.png" alt="Food Ordering Logo">
+            </div>
+            <nav class="menu">
+            <ul>
+                <li><a href="#home">HOME</a></li>
+                <li><a href="#about">ABOUT</a></li>
+                <li><a href="#service">SERVICE</a></li>
+                <li><a href="#blog">BLOG</a></li>
+                <li><a href="#contact">CONTACTS</a></li>
+                <li><a href="#contact">DASHBOARD</a></li>
+            </ul>
+            </nav>
+        </header>
+        <section class="hero">
+            <h1>Ordering food</h1>
+            <p>Quality food and best services</p>
+        </section>
+    </div>
+
+    <section class="filters">
+    <div class="filter-group"></div>
+        <select id="categoryFilter">
+            <option value="categories">Categories</option>
+            <option value="all">All</option>
+            <option value="soup">Soup</option>
+            <option value="pasta">Pasta</option>
+            <option value="pizza">Pizza</option>
+            <option value="sweets">Sweets</option>
+            <option value="drinks">Drinks</option>
+        </select>
+        <select id="sortFilter">
+            <option value="none">Sorting</option>
+            <option value="asc">Ascending</option>
+            <option value="desc">Descending</option>
+        </select>
+        <select id="dietFilter">
+            <option value="all">All</option>
+            <option value="regular">Regular</option>
+            <option value="vegetarian">Vegetarian</option>
+            <option value="vegan">Vegan</option>
+        </select>
+    </div>
+    <div class="price-slider">
+    <span id="priceValue">$5</span>
+    <input type="range" id="priceSlider" min="5" max="100" value="100">
+    <span id="priceValue">$100</span>
+</div>
+</section>
+
+<section class="products">
+    <div class="item soup" data-diet="regular">
+        <img src="./resources/images/test6.png" alt="Chicken Soup">
+        <h3>Chicken Soup</h3>
+        <p>Entrees</p>
+        <span>$13.00</span>
+        <button onclick="addToCart('Chicken Soup', 13.00)">Adaugă în coș</button>
```

Comment:
You don't need to change it now, but please write the texts in English in the future unless instructed otherwise.